export_variables.sh

```diff
@@ -5,29 +5,33 @@ private_ranch="rhel7 rhel8 rhel9 rhel9-unsubscribed"
 all_os="$public_ranch $private_ranch"
 
 os_test="$1"   # options: centos7, c9s, fedora, rhel7, rhel8, rhel9, rhel9-unsubscribed
-test_case="$2" # options: container, openshift
+test_case="$2" # options: container, openshift, openshift-4
 if [ -z "$os_test" ] || ! echo "$all_os" | grep -q "$os_test" ; then
   echo "::error::os_test '$os_test' is not valid"
   echo "::warning::choose one of: $all_os"
   exit 5
 fi
 
 # container tests vs openshift tests
-if [ -z "$test_case" ] || [ "$test_case" = "container" ] ; then
-  test_case="container"
-  tmt_plan_suffix="-docker"
-  context_suffix=""
-  test_name="test"
-elif [ "$test_case" = openshift ] ; then
-  if [ "$os_test" = "centos7" ] || [ "$os_test" = "rhel7" ] ; then
-    context_suffix=" - OpenShift 3"
-    tmt_plan_suffix="-openshift-3"
-    test_name="test-openshift-3"
-  else
-    context_suffix=" - OpenShift 4"
-    tmt_plan_suffix="-openshift4"
-    test_name="test-openshift-4"
-  fi
+if [ -z "$test_case" ] ; then
```

Comment:
This condition is not valid. If `$test_case` is not set we want to fall to "container" case. If `$test_case` is "container" we fall to "container" case too.
If `$test_case` is "openshift" or "openshift-4" we want to fall to respective cases.

Reply:
if test_case we want to fall to error. But the condition is wrong, you are right. This should be fixed by https://github.com/sclorg/tfaga-wrapper/pull/4/commits/a412952d3fe23c7a654186d2392d2eb8a6654c39.

Reply:
Yes, this seems too as a valid solution, thanks for fixing this.